test/e2e/set/scyllacluster/scyllamanager.go

```diff
@@ -132,6 +132,49 @@ var _ = g.Describe("Scylla Manager integration", func() {
 		o.Expect(repairTask.ID).To(o.Equal(sc.Status.Repairs[0].ID))
 		o.Expect(repairTask.Properties.(map[string]interface{})["parallel"].(json.Number).Int64()).To(o.Equal(int64(123)))
 
+		framework.By("Updating the repair task for ScyllaCluster")
+		sc, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Patch(
+			ctx,
+			sc.Name,
+			types.JSONPatchType,
+			[]byte(`[{"op":"replace","path":"/spec/repairs/0/parallel","value":1}]`),
+			metav1.PatchOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(sc.Spec.Repairs[0].Parallel).To(o.Equal(int64(1)))
```

Comment:
nit: please assert the opposite before the patch as well

Reply:
i suppose status assert would be useful too

Reply:
added, thanks